src/XRP/issued-currency-client.ts

```diff
@@ -389,6 +389,22 @@ export default class IssuedCurrencyClient {
     )
   }
 
+  public async freezeTrustLine(
+    accountToFreeze: string,
+    currencyName: string,
+    wallet: Wallet,
+  ): Promise<TransactionResult> {
+    return await this.sendTrustSetTransaction(
+      accountToFreeze,
+      currencyName,
+      // You can change the trust line when you freeze it, but an amount of 0
+      // would be the most conservative amount.
+      '0',
```

Comment:
so this is pretty opinionated of me. however, you would typically freeze a trust line if something sketchy happened, and the actual balance can still be reduced by the counter party sending ious back to the issuer. it didn't really feel like it made too much sense to have an exposed `amount` param when all you want to do is freeze the tl

lemme know what you think!

Reply:
seems reasonable - it appears to also be recommended

Reply:
agreed - we can change this in the future if we really have to but I think the benefits are worth it